gslib/utils/copy_helper.py

```diff
@@ -1612,7 +1617,12 @@ def _SetContentTypeFromFile(src_url, dst_obj_metadata):
               'Encountered OSError running "file -b --mime %s"\n%s' %
               (real_file_path, e))
       else:
-        content_type = mimetypes.guess_type(real_file_path)[0]
+        for extension, extension_content_type in COMMON_EXTENSION_RULES.items():
+          if real_file_path.endswith(extension):
+            content_type = extension_content_type
+            break
+        if not content_type:
+          content_type = mimetypes.guess_type(real_file_path)[0]
```

Comment:
The zero index is a magic number.
Maybe add a code comment giving insight into what `guess_type` returns or destructure to give insight into how many return values there are:
```
content_type, _, _ = mimetypes.guess_type(real_file_path)
```

Reply:
This came directly from the gsutil implementation, but now's a good time to improve it for sure!